### ISSUE TITLE
Docs version picker makes sidenav hard to use with a keyboard or screen reader

### ISSUE THREAD

**Issue by marcysutton** [angular/material#8896]
**Actual Behavior**:
 -  `What is the issue?` *
Version picker links in the docs sidenav aren't hidden from keyboard and screen reader users when the list is visually closed. That means it's still a huge pain to try to navigate the sidebar with a keyboard. This issue was open previously as https://github.com/angular/material/issues/4106, and it still exists.

 -  `What is the expected behavior?`
The `menu-toggle-list` should be hidden with `display: none` after animating completes. This could be done with CSS keyframes if it impacts the animation. Alternatively, links could be removed from the tab order with `tabindex="-1"` on each control and `aria-hidden="true"` on the parent, but that would take more work.

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:`
 -  `Details:`
Go to any page on https://material.angularjs.org and try to tab through the sidenav. Observe your keyboard focus go through every item in the (closed) version picker, and then the expand/collapse links which work correctly. 

See the version picker announced even when it is closed:
<img width="1429" alt="Version picker announced even though it is closed" src="https://cloud.githubusercontent.com/assets/1045233/16435475/21b7d5cc-3d4b-11e6-9cde-1acbe3843e45.png">

**Angular Versions**: *
 -  `Angular Version:`
 -  `Angular Material Version:`
Current / All past versions 

**Additional Information**:
 -  `Browser Type: *`
 -  `Browser Version: *`
 -  `OS: *`
 -  `Stack Traces:`
All - Visible with keyboard alone, but also with Voiceover, NVDA, JAWS, etc.
----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by marcysutton** [comment]
Yay!
